### PR TITLE
Document Data Streams Monitoring schema tracking

### PR DESCRIPTION
**Pull request by urseberry**
### What does this PR do? What is the motivation?
Document Data Streams Monitoring's new schema tracking feature.
https://datadoghq.atlassian.net/browse/DOCS-8292

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/data_streams/schema_tracking.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+title: Schema Tracking
+---
+
+{{% site-region region="gov,ap1" %}}
+<div class="alert alert-warning">
+    Data Streams Monitoring is not available for the {{< region-param key="dd_site_name" >}} site.
+</div>
+{{% /site-region %}}
+
+<div class="alert alert-warning">Schema tracking is in beta for Java services using Protobuf and Avro.</div>
```
Comment (estherk15) [inline content/en/data_streams/schema_tracking.md:11]: Optional formatting

```suggestion
<div class="alert alert-info">Schema tracking is in **beta** for Java services using Protobuf and Avro.</div>
```

File: content/en/data_streams/schema_tracking.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+Data Streams Monitoring schema tracking provides visibility into schemas used by producers and consumers, and how schema issues impact downstream services. You can track new schemas added, schemas with errors, and schema evolutions to manage schema migrations and identify issues.
+
+Changing a schema produced by a service without updating the consumer can lead to the consumer struggling to process payloads, blocking further data flow downstream. Understanding schema changes ensures data compatibility between producers and consumers, and ultimately prevents issues.
+
+## Prerequisites
+
+You must have [installed][1] Data Streams Monitoring on your Java producer and consumer services.
```
Comment (estherk15) [inline content/en/data_streams/schema_tracking.md:19]: Optional formatting
```suggestion
You must have [Data Streams Monitoring installed][1] on your Java producer and consumer services.
```

**Review by estherk15 — APPROVED** [review]
Looks good! Left optional formatting suggestion, but approved!

File: content/en/data_streams/schema_tracking.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+For each service you track in Data Streams Monitoring, you can see information about the schemas that it uses.
+
+To view schema information at the service level:
+1. Navigate to [Data Streams Monitoring][4].
+1. Ensure the **Explore** tab is selected.
+1. Click on a service. The service detail side panel appears.
+1. Select the **Schemas** tab.
+
+On the schemas tab, you can:
+- View input throughput by schema
```
Comment (urseberry) [inline content/en/data_streams/schema_tracking.md:63]:
```suggestion
- View input throughput by schema.
```

**Pull request merged**